### PR TITLE
Feature/mfmehari/krok obio

### PR DESCRIPTION
**Pull request by mfmehari**
OBIO (NOBM) coupled to GEOS radiation.
It is a zero diff PR.

### CODE REVIEW

File: GEOSsolar_GridComp/GEOS_SolarGridComp.F90
Comment (dr0cloud) [inline GEOSsolar_GridComp/GEOS_SolarGridComp.F90:1]: @mfmehari Hi Michael, looks good, but note that this code:
    if (DO_OBIO/=0) then
        SOLAR_TO_OBIO = .TRUE.
     else
        SOLAR_TO_OBIO = .FALSE.
     endif
can be more concisely written as:
SOLAR_TO_OBIO = (DO_OBIO/=0).
Peter.

Comment (mfmehari) [reply GEOSsolar_GridComp/GEOS_SolarGridComp.F90:1]: @dr0cloud Hi Peter, good I have changed it to your statement. Thank you!

**Review by dr0cloud — DISMISSED** [review]

**Review by dr0cloud — APPROVED** [review]

**Comment by mathomp4** [comment]
Tested and zero-diff.

**Comment by mfmehari** [comment]
Thank you! @mathomp4 @dr0cloud

**Pull request merged**